internal/api/genapi/templates.go

```diff
@@ -788,7 +786,7 @@ func (n {{ .Name }}ListResult) GetResponseType() string {
 }
 
 func (n {{ .Name }}ListResult) GetResponse() *api.Response {
-	return n.response
+	return n.Response
```

Comment:
Since this already returns a pointer to the underlying value... do we actually need this change? Can't we just mutate the returned value through the pointer?

Reply:
I see, and we can't dereference the pointer to assign because it's nil.